data/ipl1.cfg

```diff
@@ -26,23 +26,33 @@ APOGEE_REDUX = %(IPL_ROOT)s/spectro/apogee/redux
 # Astra paths
 mwmVisit = $MWM_ASTRA/{v_astra}/{run2d}-{apred}/spectra/visit/@cat_id_groups|/mwmVisit-{v_astra}-{cat_id}@component_default|.fits
 mwmStar = $MWM_ASTRA/{v_astra}/{run2d}-{apred}/spectra/star/@cat_id_groups|/mwmStar-{v_astra}-{cat_id}@component_default|.fits
+
 astraStarASPCAP = $MWM_ASTRA/{v_astra}/{run2d}-{apred}/results/star/@cat_id_groups|/astraStar-ASPCAP-{v_astra}-{cat_id}@component_default|-{task_id}.fits
 astraStarFERRE = $MWM_ASTRA/{v_astra}/{run2d}-{apred}/results/star/@cat_id_groups|/astraStar-FERRE-{v_astra}-{cat_id}@component_default|-{task_id}.fits
-astraStarCannon = $MWM_ASTRA/{v_astra}/{run2d}-{apred}/results/star/@cat_id_groups|/astraStar-Cannon-{v_astra}-{cat_id}@component_default|-{task_id}.fits
+astraStarTheCannon    = $MWM_ASTRA/{v_astra}/{run2d}-{apred}/results/star/@cat_id_groups|/astraStar-TheCannon-{v_astra}-{cat_id}@component_default|-{task_id}.fits
```

Comment:
Has the filenames for these products changed on disk for IPL-1?  If not, then only the template name should be changed to match IPL-2.  The template path should remain the same to correctly resolve to the real file.

Reply:
That was added in IPL1, but no files were ever actually generated.  I can move it to the IPL2 cfg if you prefer, though there are at least a few more paths in IPL1 that don't correspond to any real files.  I think Andy added them for the sake of planning.

Reply:
Thanks for clarifying.  In general I don't like cluttering the configs with paths that don't exist in that release, but I know you preferred a complete list in each config release, which is fine.  I think the rest of this is satisfactory.